src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -479,6 +468,10 @@
 helps['iot hub policy show'] = """
 type: command
 short-summary: Get the details of a shared access policy of an IoT hub.
+examples:
+-   name: Get the details of a shared access policy of an IoT hub.
+    text: az iot hub policy show --query [0] --hub-name MyHub --name MySharedAccessPolicy
+    crafted: true
```

Comment:
Again, --query [0] on a show command will output nothing. I'm not sure how this has value.